package.json

```diff
@@ -9,8 +9,8 @@
     "lib",
     "*.js"
   ],
-  "main": "./lib/index.js",
-  "module": "./index.js",
```

Comment:
is this the fix that addresses your build issue?

Reply:
I think your main path is wrong. It doesn't address to compiled code.

Reply:
Just refer below code.

https://github.com/archiverjs/node-crc32-stream/blob/master/package.json#L18